generate.go

```diff
@@ -1,4 +1,4 @@
 package falconer
 
 //go:generate gojson -input example.yaml -o config.go -fmt yaml -pkg falconer -name Config
-//go:generate protoc --proto_path=vendor:. falconer.proto --go_out=plugins=grpc:.
```

Comment:
What was the difference here? (I don't remember offhand, and I could look it up, but we should document it in the PR comments anyway.)

Reply:
It's just that the `--proto_path` (which is the same as `-I`) doesn't seem to take multiple colon-separated options sanely - that's all.